README.md

```diff
@@ -56,4 +67,24 @@ val cachingPath = cachingFilesystem.convertToCachingPath(path)
 // Continue your code using cachingPath...
 ```
 
+## Releasing:
+1. Make and checkout a release branch on github.
+2. Change the version in gradle.properties to a non-SNAPSHOT version.
+3. Update the CHANGELOG.md for the impending release.
+4. Run `git commit -am "Release AttributeCachingFileSystem X.Y.Z."` (where X.Y.Z is the new version) in the terminal or
+   command line.
+5. Make a PR with your changes.
+6. Merge the release PR after approval, tag the commit on the main branch with
+   `git tag -a X.Y.Z -m "AttributeCachingFileSystem X.Y.Z"`(X.Y.Z is the new version).
```

Comment:
```suggestion
   `git tag -a X.Y.Z -m "X.Y.Z"`(X.Y.Z is the new version).
```

Reply:
I was just following the conventions I did for my detekt extensions and for our own internal projects though

Reply:
okay i will do this